src/main/features/dashboard/views/defendant.njk

```diff
@@ -20,13 +22,13 @@
         <p>{{ t('If you don’t owe the money you can <a href="%s" target="blank">ask the court to cancel</a> the judgment.',
               { postProcess: 'sprintf', sprintf: [ 'https://www.gov.uk/county-court-judgments-ccj-for-debt/if-you-dont-owe-the-money' ]}) | safe }}</p>
       {% else %}
-        <p>{{ t('Your defence is being processed. We will email you to explain what to do next.') }}</p>
```

Comment:
@yarpo , is that removal part of the story?

Reply:
According to Julian we should display the same status on the main dashboard and here. This was very generic message that was replaced with the proper one.

Reply:
Thanks for explaining this to me.